tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -56,20 +56,20 @@ def tearDown(self) -> None:
                 os.remove(os.path.join(_DATAMODEL_FOLDER, os.path.split(data_model_module)[-1]))
 
     def test_valid_json_policy_spec(self):
```

Comment:
Could you add a similar test that validates when passing in one+ ignore_files, they don't show up in the loaded specs?